tools/extra/pac_hssi_config/pac_hssi_config.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```

Comment:
These scripts should go in a separate pull request.

Reply:
fixed , move to another pull request